validation/vortex_street_2d/run_validation.jl

```diff
@@ -0,0 +1,7 @@
+using TrixiParticles
+
+# results in [90k particles, 220k particles, 1.2M particles, 5M particles]
+for resolution_factor in [0.08, 0.05, 0.02, 0.01]
+    trixi_include(joinpath(validation_dir(), "vortex_street_2d", "vortex_street_2d.jl"),
+                  factor_D=resolution_factor, write_to_vtk=false, tspan=(0.0, 50.0))
+end
```

Comment:
Where is the validation? You want to compare the experimental with the analytical Strouhal number, no?